internal/actions/pr.go

```diff
@@ -179,6 +185,7 @@ func CreatePullRequest(ctx context.Context, repo *git.Repo, client *gh.Client, o
 	if didCreatePR {
 		action = "created"
 	} else {
+		// TODO: need to update the PR metadata possibly here
```

Comment:
what happens right now if we don't update the metadata here?

Reply:
Nothing right now since we don't use this on the server side. My plan is to finish this TODO then change the Aviator server side so that it's reading this information instead of trying to implicitly determine the shape of the stack based on how the PRs are arranged.